end-to-end-tests/wdio/webdriver-manager.conf.js

```diff
@@ -135,7 +135,15 @@ exports.config = {
     // Test reporter for stdout.
     // The only one supported by default is 'dot'
     // see also: http://webdriver.io/guide/testrunner/reporters.html
-    reporters: ['spec'],
+    reporters: ['spec', 'junit'],
```

Comment:
this is to enable reports again on circleCI

Reply:
the summary at the top of the run. kinda useful now the end to end test stuff is so bloated with selenium logs